code/modules/admin/topic.dm

```diff
@@ -1775,8 +1775,8 @@
 					H.update_mutations()
 				logmsg = "superpowers."
 		if(logmsg)
-			log_admin("[key_name(src.owner)] answered [M]'s prayer with a blessing: [logmsg]")
-			message_admins("[key_name_admin(src.owner)] answered [M]'s prayer with a blessing: [logmsg]")
+			log_admin("[key_name(src.owner)] answered [key_name_admin(M)]'s prayer with a blessing: [logmsg]")

```

Comment:
use key_name in log_admin, not key_name_admin (the former contains anchor links that look nasty in the logs)